src/Views/HtmlViewBridge.js

```diff
@@ -773,7 +775,11 @@ HtmlViewBridge.prototype.raiseServerEvent = function (eventName) {
         argumentsArray[i] = arguments[i];
```

Comment:
@acuthbert I left this as it was but I was wondering if we should leave out the callback functions from the arguments we're going to pass to the server. Client side callback functions won't serve much purpose on the server side.